packages/mds-geography/api.ts

```diff
@@ -0,0 +1,185 @@
+      } catch (err) {
+        await log.error('failed to read geography', err.stack)
+        res.status(404).send({ result: 'not found' })
+      }
+    }
+  )
+
+  app.post(
+    pathsFor('/geographies/'),
+    checkAccess(scopes => scopes.includes('policies:write')),
+    async (req, res) => {
+      const geography = req.body
+      const details = geographyValidationDetails(geography)
+
+      if (details != null) {
+        await log.error('invalid policy json', details)
+        return res.status(400).send(details)
+      }
+
+      try {
+        await db.writeGeography(geography)
+        return res.status(201).send(geography)
+      } catch (err) {
+        if (err.code === '23505') {
+          return res
+            .status(409)
+            .send({ result: `geography ${geography.geography_id} already exists! Did you mean to PUT?` })
+        }
+        /* istanbul ignore next */
+        await log.error('failed to write geography', err.stack)
+        /* istanbul ignore next */
+        return res.status(500).send(new ServerError(err))
+      }
+    }
+  )
+
+  app.put(
+    pathsFor('/geographies/:geography_id'),
+    checkAccess(scopes => scopes.includes('policies:write')),
+    async (req, res) => {
+      const geography = req.body
+      const details = geographyValidationDetails(geography)
+
+      if (details != null) {
+        await log.error('invalid policy json', details)
+        return res.status(400).send(details)
+      }
+
+      try {
+        await db.editGeography(geography)
+        return res.status(201).send(geography)
+      } catch (err) {
+        await log.error('failed to write geography', err.stack)
+        return res.status(404).send({ result: 'not found' })
+      }
+    }
+  )
+
+  app.delete(
+    pathsFor('/geographies/:geography_id'),
+    checkAccess(scopes => scopes.includes('policies:delete')),
+    async (req, res) => {
+      const { geography_id } = req.params
+      try {
+        await db.deleteGeography(geography_id)
+        return res.status(200).send({ result: `Successfully deleted geography of id ${geography_id}` })
+      } catch (err) {
+        await log.error('failed to delete geography', err.stack)
+        return res.status(404).send({ result: 'geography either not found or already published' })
+      }
+    }
+  )
+
+  app.get(
+    pathsFor('/geographies/:geography_id/meta'),
+    checkAccess(scopes => scopes.includes('policies:read')),
+    async (req, res) => {
+      const { geography_id } = req.params
+      try {
+        const geography_metadata = await db.readSingleGeographyMetadata(geography_id)
+        return res.status(200).send(geography_metadata)
+      } catch (err) {
+        await log.error('failed to read geography metadata', err.stack)
+        return res.status(404).send({ result: 'not found' })
+      }
+    }
+  )
+
+  app.get(
+    pathsFor('/geographies'),
+    checkAccess(scopes => scopes.includes('policies:read')),
```

Comment:
Either need a new scope, or none at all (don't know if we need to block off geography reading at all)